package.json

```diff
@@ -77,6 +77,15 @@
         "title": "API Repository from Postman JSON",
         "when": "false"
       },
+      {
+        "command": "fluttergpt.createCodeFromDescription",
+        "title": "FlutterGPT Create: From Description"
+      },
+      {
+        "command": "fluttergpt.createCodeFromDescription.menu",
```

Comment:
Register this in `fluttergpt.createMenu` as well so the command is also accessible via right-click menu under FlutterGPT: Create section.